### PR TITLE
BUG: Store SchemaUpdateAutoCreateDirectory for DICOM browser in settings

### PR DESCRIPTION
**Pull request by cpinter**
This fixes automated tests that time out because of the popup that appears.

### CODE REVIEW

**Comment by cpinter** [comment]
Based on https://discourse.slicer.org/t/update-on-build-factory-status/8146

File: Libs/DICOM/Widgets/ctkDICOMBrowser.h [hunk 1]
```diff
@@ -107,14 +107,14 @@ class CTK_DICOM_WIDGETS_EXPORT ctkDICOMBrowser : public QWidget
...
   /// Since the summary dialog is modal, we give the option of disabling it for batch modes or testing.
   void setDisplayImportSummary(bool);
-  bool displayImportSummary();
+  bool displayImportSummary()const;
   /// Option to show dialog to confirm removal from the database (Remove action). Off by default.
   void setConfirmRemove(bool);
-  bool confirmRemove();
+  bool confirmRemove()const;
   /// Option to determine whether the new database folder is automatically created or set by the user in a popup.
   /// Automatically created folder will be ../[CurrentDatabaseFolderName]-[NewSchemaVersion]. Off by default.
```
Comment (lassoan) [inline Libs/DICOM/Widgets/ctkDICOMBrowser.h]: it seems that it is On by default now

Comment (cpinter) [reply Libs/DICOM/Widgets/ctkDICOMBrowser.h]: Right. If we have a decision on the default I'll update it here too.

**Comment by jcfr** [comment]
@cpinter Assuming the user set `DICOM/SchemaUpdateOption=NeverUpdate`, is there anything to be changed with `DICOM/ShemaUpdateAutoCreateDirectory` ? 

Since @dzenanz would like to use a dicom database with both Slicer 4.10 and the nightly, I was wondering if this use case was supported.

**Comment by dzenanz** [comment]
`NeverUpdate` still shows the dialog "Choose existing database / Select empty folder for new DICOM database". Clicking "Cancel" twice dismisses it. Entering DICOM module requires 2 more cancels, and then the current database is shown. Not ideal, but way better than having to wait for the database to be converted to the new format.

**Comment by jcfr** [comment]
> still shows the dialog "Choose existing database / Select empty folder for new DICOM database". Clicking "Cancel" twice dismisses it.

This fix from @Sunderlandkyl  integrated this morning should help: https://github.com/Slicer/Slicer/commit/c0c5df6c4f8461339cf2ae2d741dcf420dceed65

**Comment by cpinter** [comment]
> Assuming the user set DICOM/SchemaUpdateOption=NeverUpdate, is there anything to be changed with DICOM/ShemaUpdateAutoCreateDirectory ?

If the setting is NeverUpdate, then it means that the user does not care about the old database but always wants an empty one instead of the one with the old schema. I think auto create directories still makes sense in that case.

> Since @dzenanz would like to use a dicom database with both Slicer 4.10 and the nightly, I was wondering if this use case was supported.

Yes, both Slicer versions can be used without changing the settings before each run.

**Comment by lassoan** [comment]
This pull request has been superseded by DICOM database rework.

**Pull request closed without merging**